### PR TITLE
[StructuralMechanicsApplication] Missing second create method on elements and conditions

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D4N.cpp [hunk 1]
```diff
@@ -137,7 +137,7 @@ ShellThickElement3D4N::MITC4Params::MITC4Params(const ShellQ4_LocalCoordinateSys
     Cy = - LCS.Y1() - LCS.Y2() + LCS.Y3() + LCS.Y4();
 
     double Alpha = std::atan( Ay / Ax );
-    double Beta  = 3.141592653589793 * 0.5 - std::atan( Cx / Cy );
+    double Beta  = Globals::Pi * 0.5 - std::atan( Cx / Cy );
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D4N.cpp:140]: :+1: 
Lets hope this doesn't change the values of the tests :D

File: applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp [hunk 2]
```diff
@@ -39,6 +39,15 @@ TrussElementLinear3D2N::Create(IndexType NewId,
       NewId, rGeom.Create(rThisNodes), pProperties);
 }
 
+Element::Pointer
+TrussElementLinear3D2N::Create(IndexType NewId,
+                            GeometryType::Pointer pGeom,
+                               PropertiesType::Pointer pProperties) const {
+  const GeometryType &rGeom = this->GetGeometry();
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp]: is this intended?
It is not used...

**Review by philbucher — COMMENTED** [review]
Nice, thanks a lot

(FYI @MFusseder I think you told me you would need tis for sth)

File: applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp (hunk 2 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_elements/truss_element_linear_3D2N.cpp]: Copy paste...

**Review by philbucher — APPROVED** [review]
thx!

**Pull request merged**